src/styles/_price.scss

```diff
@@ -0,0 +1,25 @@
+@use './variables' as *;
+@use './mixins' as *;
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  width: $primary-width;
+  height: 18px;
```

Comment:
[checklist](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules): Don't fix container size (if there is no such a requirement)
Let the content size dictate it. To avoid overflow or accidental scroll bar